Src/DeUrgenta.Admin.Api/DeUrgenta.Admin.Api.csproj

```diff
@@ -6,11 +6,13 @@
 	<PropertyGroup>
 		<GenerateDocumentationFile>true</GenerateDocumentationFile>
 		<NoWarn>$(NoWarn);1591</NoWarn>
+		<UserSecretsId>7f5aac43-b876-4147-9f5b-5a8cff439332</UserSecretsId>
```

Comment:
I do not understand where this is used

Reply:
This is to configure your local env without changing appsettings.json. That way you don't have to worry about committing it accidentally https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-5.0&tabs=windows